### ISSUE TITLE
PowerShell Core 7.0.0-rc2 loads with profile.

### ISSUE THREAD

**Issue by casz** [gerardog/gsudo#10]
When launching gsudo from powershell core v7.0.0-rc2 it launches with profile.

See my profile here: https://github.com/casz/dotfiles/blob/master/powershell/profile.ps1

![image](https://user-images.githubusercontent.com/1661688/73100548-0ed1f200-3eee-11ea-9e89-083d381167d0.png)

Of course I could remove my "Modules Loaded" message 🤔 But it seems that it also adds two new lines in pwsh rather than just one newline.

**Comment by gerardog** [comment]
Can you please try `gsudo config PowerShellCore7Arguments "-NoProfile -Command"`?

**Comment by casz** [comment]
Yup that fixes the behavior :)

**Comment by casz** [comment]
However the default settings for Windows Powershell seems to be `-NoProfile`

https://github.com/gerardog/gsudo/blob/c277d7a846657a5c7167817ed07d1d1140c22a42/src/gsudo/GlobalSettings.cs#L24-L26

**Comment by gerardog** [comment]
Thanks @casz, with your help I think I figured out all cases.

One supported gsudo feature is that if you just run `gsudo` it elevates the current shell.
This has different meanings per shell:
=> PowerShell => powershell --NoLogo 
=> PowerShellCore => pwsh --NoLogo 
=> PowerShell 6.0 to 6.2.3 with GlobalInstall => pwsh  ( because pwsh --NoLogo  throws)
This scenario was broken with #11 for the global install.
When elevating the shell, IMHO is preferred to load the profile.

And for `gsudo {command}`, no profile load.
=> PowerShell => powershell --NoLogo -NoProfile -Command {command} 
=> PowerShellCore => pwsh --NoLogo -NoProfile -Command {command}
=> PowerShell6GlobalInstall => pwsh {command} (shows a warning "update to PsCore>=6.2.4" and then loads the profile. Sorry, the best that one could possibly do).

Given all possibilities, I removed the configurability. The main reason for the setting being there in the first place was that at that moment I knew there were some case I was not yet aware of.

Let me know if release v0.6 works well for you.
